schemas/applications/schema/buttons.schema.json

```diff
@@ -21,6 +21,30 @@
         },
         "unload": {
           "$ref": "events/unload.schema.json"
+        },
+        "transition": {
+          "$ref": "events/transition.schema.json"
+        },
```

Comment:
this one makes no sense for the button and can be removed. 

Transition is designed as a shortcut between components. If component A triggers a transition to component B, then it will unload component A and load component B. But, if a button is connected to component B, it cannot "transition" since we cannot "unload" the button.
```suggestion
```